src/hive/__tests__/e2e.hive.test.ts

```diff
@@ -0,0 +1,110 @@
+import { IMerjoonProjects, IMerjoonTasks, IMerjoonUsers } from '../../common/types';
+import { HiveService } from '../service';
+import { getHiveService } from '../hive-service';
+import { ID_REGEX } from '../../utils/regex';
+
+describe('e2e Hive', () => {
+  let service: HiveService;
+
+  beforeEach(async () => {
+    service = await getHiveService();
+  });
+
+  it('getUsers', async () => {
+    const users: IMerjoonUsers = await service.getUsers();
+
+    expect(Object.keys(users[0])).toEqual(expect.arrayContaining([
+      'id',
+      'remote_id',
+      'name',
+      'email_address',
+      'created_at',
+      'modified_at',
+    ]));
+
+    expect(users[0]).toEqual({
+      id: expect.stringMatching(ID_REGEX),
+      remote_id: expect.any(String),
+      name: expect.any(String),
+      email_address: expect.any(String),
+      created_at: expect.any(Number),
+      modified_at: expect.any(Number),
+    });
+  }, 100000);
+
+  it('getProjects', async () => {
+    const projects: IMerjoonProjects = await service.getProjects();
+
+    expect(Object.keys(projects[0])).toEqual(expect.arrayContaining([
+      'id',
+      'remote_id',
+      'name',
+      'description',
+      'remote_created_at',
+      'remote_modified_at',
+      'created_at',
+      'modified_at',
+    ]));
+
+    expect(projects[0]).toEqual({
+      id: expect.stringMatching(ID_REGEX),
+      remote_id: expect.any(String),
+      name: expect.any(String),
+      description: expect.any(String),
+      remote_created_at: expect.any(Number),
+      remote_modified_at: expect.any(Number),
+      created_at: expect.any(Number),
+      modified_at: expect.any(Number),
+    });
+  }, 100000);
+
+  it('getTasks', async () => {
+    const tasks: IMerjoonTasks = await service.getTasks();
+
+    expect(Object.keys(tasks[103])).toEqual(expect.arrayContaining([
+      'id',
+      'remote_id',
+      'name',
+      'assignees',
+      'status',
+      'description',
+      'projects',
+      'remote_created_at',
+      'remote_modified_at',
+      'created_at',
+      'modified_at',
+    ]));
+
+    expect(tasks[103]).toEqual({
```

Comment:
change the task[0] in hive in a way that all tests with task[0] pass here, to keep things constant